frontend/src/host_orchestrator/main.go

```diff
@@ -56,39 +54,13 @@ func startHttpServer(addr string, port int) error {
 	return http.ListenAndServe(fmt.Sprintf("%s:%d", addr, port), nil)
 }
 
-func startHttpsServer(addr string, port int, certPath string, keyPath string) error {
```

Comment:
Can we keep this functionality? Being able to run https is not a bad idea, even if we don't use it at the moment.

Reply:
Done, also kept the logic of starters, we we can start multiple https servers.